app/web/features/auth/notifications/PushNotificationDenied.tsx

```diff
@@ -0,0 +1,74 @@
+import Alert from "components/Alert";
+import { useTranslation } from "i18n";
+import { AUTH } from "i18n/namespaces";
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  alert: {
+    marginBottom: theme.spacing(3),
+    marginTop: theme.spacing(2),
+  },
+}));
+
+const PushNotificationDenied = () => {
+  const { t } = useTranslation(AUTH);
+  const userAgent = navigator.userAgent.toLowerCase();
+  const classes = useStyles();
+
+  const getBrowserInstructions = () => {
+    if (userAgent.includes("chrome")) {
```

Comment:
I don't think this actually tells you the truth... Everything pretends for instance to be compatible with firefox. Again, can be sorted out later.

Reply:
I was using Chrome and it was working for me, but it's true I haven't tried all browsers. Can make a ticket to thoroughly test them for different ones.

Reply:
Yeah we can probably use some lib, but yeah let's add an issue and do it later!